lib/block_view_message.go

```diff
@@ -825,6 +849,71 @@ func (bav *UtxoView) _connectMessagingGroup(
 		}
 	}
 
+	// V3 Messages: MUTING and UNMUTING members
+	if blockHeight >= bav.Params.ForkHeights.DeSoV3MessagesBlockHeight {
+		if existingEntry != nil && !existingEntry.isDeleted {
+			value, operationTypeExists := txn.ExtraData["OperationType"]
```

Comment:
nit: Can we make "OperationType" a constant? We usually put ExtraData constants towards the bottom of constants.go

Reply:
good catch, done